### PR TITLE
Improvements to interface

### PR DESCRIPTION
**Pull request by ashmaroli**
* add module name to version `<span>`
* blend logo, sidebar-routes and header
* add icons for file to maintain consistency
* Prepend collection-label with `Collections: ` [debatable]

intefc

### CODE REVIEW

**Comment by mertkahyaoglu** [comment]
I must say there is too much darkness and thickness here that makes me feel bad :trollface:

**Comment by ashmaroli** [comment]
> there is too much darkness and thickness here

thickness?
would reverting the header to white be sufficient?

**Comment by mertkahyaoglu** [comment]
> thickness

I meant bold fonts. Maybe it's just me but really catching my eyes

**Comment by ashmaroli** [comment]
okay.. I'll revert the header to white and reduce the weight on `version`..

File: src/styles/header.scss [hunk 1]
```diff
@@ -2,14 +2,14 @@
   display: flex;
   align-items: center;
   justify-content: space-between;
-  height: 75px;
-  background-color: white;
+  height: $header-height;
+  background-color: $header-color;
+  border-bottom: 1px solid #dedede;
   padding: 0 40px;
-  @include box-shadow(0 2px 4px 0 rgba(204, 204, 204, 0.4));
```
Comment (mertkahyaoglu) [inline src/styles/header.scss]: Let's keep these attributes as they are to be compatible with docs site. It depends but I really love the current design. We can get some feedback from the community to change it.

File: src/components/Editor.js [hunk 2]
```diff
@@ -34,6 +34,7 @@ class Editor extends Component {
         highlightActiveLine={false}
         className="config-editor"
         fontSize={14}
+        scrollMargin={[15, 15, 15, 15]}
```
Comment (ashmaroli) [inline src/components/Editor.js]: this is to simulate padding in the `ace-editor` module. It is activated by an `onScroll` call

Comment (ashmaroli) [inline src/components/Editor.js]: Let me know your decision on this.. try it out locally if you can.. I'll remove the rejected commits and update the branch accordingly..

**Comment by ashmaroli** [comment]
closing in favor of #308

**Pull request closed without merging**